### PR TITLE
Us018 debug mode

### PR DESCRIPTION
**Pull request by ixi150**
F1 - on/off FPS
F2 - no/off Collisions

### CODE REVIEW

File: src/mechanics/ColliderCircle.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	}
+	
+	@Override
+	public boolean collides(ColliderPoint with) 
+	{
+		return this.collides( new ColliderCircle(with.owner) );
+	}
+
+	@Override
+	public boolean collides(ColliderRectangle with)
```
Comment (EIemos) [inline src/mechanics/ColliderCircle.java]: WTF

**Comment by EIemos** [comment]
APPROVE

**Comment by grzesiek8** [comment]
APPROVE

**Comment by Quedin** [comment]
APPROVE

**Comment by Soltyskrys** [comment]
APPROVE

**Comment by Numbie** [comment]
approve

**Comment by McjMzn** [comment]
Approve

**Pull request merged**